keycloak/keycloak-operator
# [minor] Check generated code in CI

## Additional Information
Setting up the tools for compiling this repository I found out that some generated sources have(probably) been formatted with a newer version of `gofmt`.
A few tweaks enable people to use `golang` coming from `asdf` as well, instead of installing globally Go 1.13.

## Verification Steps

The steps to verify that the generated code doesn't change the diff is implemented as a CI step

## Checklist:
